Added Python Virtual Environments Folder - fresh PR

SundareshSankaran opened the pull request:
# Please include answers to these questions as part of your pull request

In the GitHub webUI, use the Write tab to modify the Markdown text that is part of the pull request. For each question simply place an **X** inside the square brackets, \[X\], that represents your answer. Using the Preview tab you can see the formatted/rendered version of the message.

- **Q1: Confirm that you have the right to submit the code that is being contributed. Please consider the origin of your code and confirm you have the appropriate rights to make the submission subject to the Apache 2.0 license that applies to everything in this repository of custom steps. If so, follow the instructions for the [Contributor Agreement](ContributorAgreement.txt) (which is based on the industry-standard Developer Certificate of Origin (DCO)).**
   - [**X** ] Yes, I have the right to submit the contributed code on behalf of myself, my company, or any other owner of the code. I have also attached my signed copy of the DCO to this message.
   - [ ] No
- **Q2: Confirm that your contribution does not include any personally identifiable information (PII), for example, in any examples used in your README file.**
   - [**X** ] My contribution does NOT include PII data
   - [ ] My contribution includes PII data
- **Q3: Confirm your contribution does not include any encryption or other export-controlled technology.**
   - [**X** ] My contribution does NOT contain encryption or other export-controlled technology
   - [ ] My contribution includes encryption or other export-controlled technology

SundareshSankaran commented:
[ContributorAgreement.txt](https://github.com/sassoftware/sas-studio-custom-steps/files/9811970/ContributorAgreement.txt)

SundareshSankaran commented:
Added a Signed Contributor Agreement above.

snlwih commented:
@SundareshSankaran , please remove the contributor agreement file from the files in your pull request. That file should not be part of the files you contribute, it should be part of the message that is associated with your pull request.

SundareshSankaran commented:
@snlwih , summary of changes:
1. I have deleted the Synthetic Data Generations folder - this will come in a fresh PR
2. I have renamed the steps as per your suggested standard (done by deleting existing and uploading the same renamed)
3. The contributor agreement, as mentioned earlier, has been removed.

snlwih commented:
SAS Internal Due Diligence Review - COMPLETED on 21OCT2022
